### PR TITLE
FileUploader: input should get the 'title' attribute with a correct value for hint working in FireFox (T1187412)

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/devextreme/js/ui/file_uploader.js [hunk 1]
```diff
@@ -301,6 +301,7 @@ class FileUploader extends Editor {
         this._$fileInput.prop({
             multiple: this.option('multiple'),
             accept: this.option('accept'),
+            title: this.option('hint') || null,
```
Comment (ksercs) [inline packages/devextreme/js/ui/file_uploader.js:304]: Why should we write `|| null` here?

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js [hunk 2]
```diff
@@ -239,3 +239,14 @@ QUnit.module('the \'name\' option', () => {
     });
 });
 
+QUnit.module('the \'hint\' option', () => {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js]: It's more readable to use `"` inside `'` to get rid of slashes

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js [hunk 3]
```diff
@@ -239,3 +239,14 @@ QUnit.module('the \'name\' option', () => {
...
+    QUnit.test('widget input should get the \'title\' attribute with a correct value for hint working in FireFox (T1187412)', function(assert) {
+        const hint = 'CUSTOM HINT';
+        const $element = $('#fileuploader').dxFileUploader({
+            hint
+        });
+        const $input = $element.find('.' + FILEUPLOADER_INPUT_CLASS);
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js]: we usually use a string literal as a more readable option
```suggestion
        const $input = $element.find(`.${FILEUPLOADER_INPUT_CLASS}`);
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js [hunk 4]
```diff
@@ -239,3 +239,14 @@ QUnit.module('the \'name\' option', () => {
...
+
+        assert.equal($input.attr('title'), hint, 'the input \'title\' attribute has correct value');
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js]: We usually use `strictEqual` everywhere where it's possible. U don't use `==` instead if `===`, right?
```suggestion
        assert.strictEqual($input.attr('title'), hint, 'the input "title" attribute has correct value');
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js [hunk 5]
```diff
@@ -239,3 +239,14 @@ QUnit.module('the \'name\' option', () => {
...
+    });
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/fileUploader.markup.tests.js:257]: Let's also add a test for "hint" option runtime change

File: packages/devextreme/js/ui/file_uploader.js (hunk 1 above)
Comment (iBat) [reply packages/devextreme/js/ui/file_uploader.js:304]: For consistence with https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/ui/widget/ui.widget.js#L158

Comment (ksercs) [reply packages/devextreme/js/ui/file_uploader.js:304]: I believe we should remove it or use nullish coalescing here. But ok, let's postpone it

**Review by ksercs — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**